<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Installer</class>
 <widget class="QDialog" name="Installer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Installer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/getrasplex.png</normaloff>:/icons/getrasplex.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;GetRasplex &lt;/span&gt;&lt;/p&gt;&lt;p&gt;Official graphical installer for the &lt;a href=&quot;http://rasplex.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;rasplex&lt;/span&gt;&lt;/a&gt; project. &lt;/p&gt;&lt;p&gt;Brought to you by Patrik and Dale.&lt;/p&gt;&lt;p&gt;Special thanks to &lt;a href=&quot;http://sourceforge.net/projects/win32diskimager/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Win32Diskimager&lt;/span&gt;&lt;/a&gt; project.&lt;/p&gt;&lt;p&gt;Icon made by &lt;a href=&quot;http://www.freepik.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Freepik&lt;/span&gt;&lt;/a&gt; from &lt;a href=&quot;http://www.flaticon.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;www.flaticon.com&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;http://donate.rasplex.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Please Donate!&lt;/span&gt;&lt;/a&gt; to support the project and&lt;br/&gt;future development : )&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="logo">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/icons/getrasplex.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="deviceSelectText">
     <property name="text">
      <string>Step 1: Select your Raspberry Pi model, and the RasPlex version to use</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="deviceSelectBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="releaseLinks"/>
     </item>
     <item>
      <widget class="QToolButton" name="linksButton">
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/refresh62.png</normaloff>:/icons/refresh62.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="releaseNotes"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Step 2: Download image or select a local image file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="downloadProgressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1">
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelDownloadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>C&amp;ancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Select &amp;image file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="removableDevicesLabel">
     <property name="text">
      <string>Step 3: Select flash target (e.g. your SD card)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="removableDevicesComboBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="refreshRemovablesButton">
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/refresh62.png</normaloff>:/icons/refresh62.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Step 4: Write image to target</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="videoGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Video output (coming soon)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="hdmiOutputButton">
        <property name="text">
         <string>Use HDMI Digital Output (default)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sdtvOutputButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Use SDTV Analog Output (coming soon)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="sdtvModeGroup">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="sdtvMode_0">
             <property name="toolTip">
              <string>Normal NTSC</string>
             </property>
             <property name="text">
              <string>Normal NTSC</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sdtvMode_1">
             <property name="toolTip">
              <string>Japanese version of NTSC - no pedestal</string>
             </property>
             <property name="text">
              <string>Japanese NTSC</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sdtvMode_2">
             <property name="toolTip">
              <string>Normal PAL</string>
             </property>
             <property name="text">
              <string>Normal PAL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sdtvMode_3">
             <property name="toolTip">
              <string>Brazilian version of PAL - 525/60 rather than 625/50, different subcarrier</string>
             </property>
             <property name="text">
              <string>Brazilian PAL</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="writeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Write SD card</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelFlashButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Can&amp;cel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="flashProgressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="messageBar">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sdtvOutputButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>sdtvModeGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
